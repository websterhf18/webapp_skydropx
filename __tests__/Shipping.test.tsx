/**
 * @format
 * @jest-environment jsdom
 */
import React from "react";
import renderer from "react-test-renderer"; // Note: test renderer must be required after react-native.
import ShippingForm from "../components/HomePage/Shipping.form";

it("Renders the Shipping Form Component", () => {
  const tree = renderer.create(<ShippingForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
