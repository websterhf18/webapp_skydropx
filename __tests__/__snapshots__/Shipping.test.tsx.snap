// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Shipping Form Component 1`] = `
<form
  className="needs-validation m-3"
>
  <div
    className="row g-3"
  >
    <div
      className="col-sm-6"
    >
      <label
        className="form-label"
        htmlFor="firstName"
      >
        Codigo postal origen
      </label>
      <input
        className="form-control"
        id="firstName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid first name is required.
      </div>
    </div>
    <div
      className="col-sm-6"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Codigo postal destino
      </label>
      <input
        className="form-control"
        id="lastName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid last name is required.
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Weight
      </label>
      <input
        className="form-control"
        id="lastName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid last name is required.
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Height
      </label>
      <input
        className="form-control"
        id="lastName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid last name is required.
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Width
      </label>
      <input
        className="form-control"
        id="lastName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid last name is required.
      </div>
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Length
      </label>
      <input
        className="form-control"
        id="lastName"
        placeholder=""
        type="text"
        value=""
      />
      <div
        className="invalid-feedback"
      >
        Valid last name is required.
      </div>
    </div>
  </div>
  <hr
    className="my-4"
  />
  <button
    className="w-100 btn btn-primary btn-lg"
    type="submit"
  >
    Enviar
  </button>
</form>
`;
